packaging/scripts/mongodb-backup_builder.sh

```diff
@@ -170,7 +170,11 @@ install_deps() {
 
     if [ "x$OS" = "xrpm" ]; then
         yum clean all
-        yum -y install epel-release git wget
+        INSTALL_LIST="epel-release git wget"
+        if [ "x$OS_NAME" = "xamzn2023" ]; then
```

Comment:
https://github.com/percona/percona-backup-mongodb/blob/dev/packaging/scripts/mongodb-backup_builder.sh#L130
in this line we detect OS so we can use RHEL variable and compare it with 2023

Reply:
Done